### PR TITLE
Added models, with incomplete setup ActiveAdmin

### PR DESCRIPTION
**Pull request by hawkprovvv**
In ActiveAdmine not can complete setup with Sorcery yet...

### CODE REVIEW

File: config/locales/devise.en.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
```
Comment (slenderock) [inline config/locales/devise.en.yml]: delete this file

File: db/schema.rb [hunk 2]
```diff
@@ -10,11 +10,37 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_02_22_173654) do
+ActiveRecord::Schema.define(version: 2019_02_24_144228) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "active_admin_comments", force: :cascade do |t|
```
Comment (slenderock) [inline db/schema.rb]: thus changes shouldn't be here

File: test/models/admin_user_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (slenderock) [inline test/models/admin_user_test.rb]: remove!

File: test/fixtures/admin_users.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (slenderock) [inline test/fixtures/admin_users.yml]: remove!

File: config/environments/development.rb [hunk 5]
```diff
@@ -58,4 +58,6 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  #config.action_mailer.default_url_options = { :host => 'localhost:3000' }
```
Comment (slenderock) [inline config/environments/development.rb]: remove this line

File: app/models/post.rb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to user
```
Comment (slenderock) [inline app/models/post.rb]: Syntax error! `:` is missing
belongs_to :user

**Review by slenderock — CHANGES_REQUESTED** [review]

File: app/models/post.rb (hunk 6 above)
Comment (hawkprovvv) [reply app/models/post.rb]: fixed

File: config/environments/development.rb (hunk 5 above)
Comment (hawkprovvv) [reply config/environments/development.rb]: fixed

File: test/fixtures/admin_users.yml (hunk 4 above)
Comment (hawkprovvv) [reply test/fixtures/admin_users.yml]: fixed

File: test/models/admin_user_test.rb (hunk 3 above)
Comment (hawkprovvv) [reply test/models/admin_user_test.rb]: fixed

File: config/locales/devise.en.yml (hunk 1 above)
Comment (hawkprovvv) [reply config/locales/devise.en.yml]: fixed

File: app/models/follower_user.rb [hunk 7]
```diff
@@ -0,0 +1,4 @@
+class FollowerUser < ApplicationRecord
+  belongs_to :user
+  belongs_to :follower, foreign_key: { to_table: :users }
```
Comment (slenderock) [inline app/models/follower_user.rb]: class_name: 'User'

Comment (slenderock) [reply app/models/follower_user.rb]: foreign_key: { to_table: :users }
doesn't works in models

File: app/models/role.rb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+class Role < ApplicationRecord
+  extend Enumerize
+
+  belongs_to :user
+
+  enumerize :name, in: [:admin, :member], default: :member
```
Comment (slenderock) [inline app/models/role.rb]: change to -> `%i[admin member]`

**Review by slenderock — CHANGES_REQUESTED** [review]
remove DS_Store files from index

**Review by slenderock — APPROVED** [review]

**Pull request merged**